.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
   - govendor sync
 
 script:
-  - go test -race -coverprofile=profile.out -covermode=atomic
+  - go test -race -coverprofile=profile.out -covermode=atomic ./...
```

Comment:
This doesn't work for sub packages for go version < 1.10.x, instead you should do it individually per package

https://github.com/codecov/example-go#caveat-multiple-files